objects/screen.c

```diff
@@ -1016,6 +1016,57 @@ luaA_screen_count(lua_State *L)
     return 1;
 }
 
+/** Add a fake screen.
+ * @tparam integer x X-coordinate for screen.
+ * @tparam integer y Y-coordinate for screen.
+ * @tparam integer width width for screen.
+ * @tparam integer height height for screen.
+ * @return The new screen.
+ * @function fake_add
+ */
+static int
+luaA_screen_fake_add(lua_State *L)
```

Comment:
Could there be a fake_resize too? It would be useful to be able to split Xephyr screen into "virtual sub screens"[1] for the tests (or use the Xinerama support and just virtually enable and disable the other screens. Also, this is actually useful for people using buggy hardware (like the Radeon HD5450[2]), MOSAIC or other unsupported ways to define screens.

[1] Virtually resize the real screen in 2, then fake_add the second part of the real screen as an imaginary second screen.
[2] That one support 3 screens per card, but expose them as 2 screens because of an (hardware?) bug.